alien/standalone/src/core/alien/kernels/simple_csr/algebra/alien_cblas.h

```diff
@@ -1,29 +1,15 @@
+﻿/// -*- tab-width: 2; indent-tabs-mode: nil; coding: utf-8-with-signature -*-
+//-----------------------------------------------------------------------------
+// Copyright 2000-2023 CEA (www.cea.fr) IFPEN (www.ifpenergiesnouvelles.com)
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: Apache-2.0
 #pragma once
 
-#ifdef __cplusplus
-extern "C" {
-#endif // __cplusplus
-
-float cblas_sdot(const int n, const float* x, const int incx, const float* y,
-                 const int incy);
-double cblas_ddot(const int n, const double* x, const int incx, const double* y,
-                  const int incy);
-void cblas_saxpy(const int n, const float alpha, const float* x, const int incx,
-                 float* y, const int incy);
-void cblas_daxpy(const int n, const double alpha, const double* x,
-                 const int incx, double* y, const int incy);
-void cblas_scopy(const int n, const float* x, const int incx, float* y,
-                 const int incy);
-void cblas_dcopy(const int n, const double* x, const int incx, double* y,
-                 const int incy);
-void cblas_sscal(const int n, const float alpha, float* x, const int incx);
-void cblas_dscal(const int n, const double alpha, double* x, const int incx);
-double cblas_dasum(const int n, const double* x, const int incx);
-double cblas_dnrm2(const int n, const double* x, const int incx);
-
-#ifdef __cplusplus
-}
-#endif // __cplusplus
```

Comment:
Might be useful for other BLAS implementations. 
Did you have conflicts with MKL declarations before ?

Reply:
no the conflict appears with with mkl from 2021b toolchain